uber/configspec.ini

```diff
@@ -446,6 +446,13 @@ guest_badge     = string(default="Guest")
 one_day_badge   = string(default="One Day")
 __many__ = string
 
+[[badge_status]]
```

Comment:
It may not be a problem for the program itself, but if I wanted to do a search for everywhere I check for a "Completed" badge status then I'd definitely want it to be a unique name. That's why I named them like this, even though it's a bit more verbose.